### PR TITLE
NFV OVS DPDK document issues fix

### PR DESCRIPTION
**Pull request by Jaganathancse**
This change is to fix NFV OVS DPDK document issues.

### CODE REVIEW

File: nfv_ovs_dpdk.md [hunk 1]
```diff
@@ -254,11 +254,11 @@ oc kustomize --load-restrictor LoadRestrictionsNone dataplane-operator/examples/
 Custom `OpenStackDataPlaneService` called `nova-custom-ovsdpdk`
 has been created as described in
 the [documentation to configure OpenStack to use OVS DPDK](ovs_dpdk.md).
-The `nova-custom-dpdk` can be seen in the
+The `nova-custom-ovsdpdk` can be seen in the
```
Comment (abays) [inline nfv_ovs_dpdk.md:257]: Maybe a nit.  Looks like the suffix might be slightly different [1] (`-ovs-dpdk`).

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/c70f4334ddbf7b87a566245e95708feaef64752b/examples/ovs_dpdk/kustomization.yaml#L5

**Review by jpodivin — CHANGES_REQUESTED** [review]
ditto like @abays

File: nfv_ovs_dpdk.md (hunk 1 above)
Comment (Jaganathancse) [reply nfv_ovs_dpdk.md:257]: done

**Review by abays — APPROVED** [review]
/lgtm

**Review by jpodivin — APPROVED** [review]

**Pull request merged**